Automattic/jetpack
# Untangle: Enable user data fields & color scheme selector

Follow-up: https://github.com/Automattic/jetpack/pull/35235

## Proposed changes:
Removes all wpcom related data from Profile and make the Admin Color Schema work when enable it by using this PR: https://github.com/Automattic/wpcomsh/pull/1669

| Before | After |
| --- | --- |
| ![image](https://github.com/Automattic/jetpack/assets/402286/46df5b57-a5a7-4d24-a1b0-f2e91fa6447f) | ![image](https://github.com/Automattic/jetpack/assets/402286/00231820-43b4-4698-8a1b-1c16204e65fb) |
| ![image](https://github.com/Automattic/jetpack/assets/402286/5fb9b856-d0cf-43cc-99e7-96979030b44c) | ![image](https://github.com/Automattic/jetpack/assets/402286/584ca303-7896-4eda-8e77-71efe42b442c) |

## Jetpack product discussion
paYJgx-4uN-p2#comment-4524

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Apply this wpcomsh PR (follow the instructions): https://github.com/Automattic/wpcomsh/pull/1669
* Set your `Admin interface style` to `Classic (wp-admin)` in `wordpress.com/hosting-config/site_slug`
* Go to `wp-admin`
* `Users` > `Profile` should link to wp-admin profile page
* Color Scheme should show different Schemes, and you can switch them.
* Try reloading after changing the Scheme.
* User data (Name, About yourself) is editable.
* Try editing it, saving and reloading.
* Color and user data should not affect Calypso/wpcom.
